### PR TITLE
updating repo link

### PR DESCRIPTION
**Pull request by sjletcher**

### CODE REVIEW

File: how-to-docs/namespace-how-to.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## How do I create a namespace in the UK cluster?
-Please visit the [ACP-Kube-Namespaces](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube-namespaces) repo on Gitlab to create a namespace.
+Please visit the [ACP-Kube](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube) repo on Gitlab to create a namespace.
 
 + Clusters are defined by the folders within the `namespaces/` directory. This directory contains all defined namespaces for each given cluster.
```
Comment (KashifSaadat) [inline how-to-docs/namespace-how-to.md]: "defined by the folders within the `policies/` directory, with each cluster having a subdirectory labelled `namespaces`" - Or something along those lines

File: how-to-docs/namespace-how-to.md [hunk 2]
```diff
@@ -19,5 +19,5 @@ metadata:
 ```
 to be defined in `namespaces/bar/foo.yaml`
```
Comment (KashifSaadat) [inline how-to-docs/namespace-how-to.md]: `policies/bar/namespaces/foo.yaml`

File: how-to-docs/namespace-how-to.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ## How do I create a namespace in the UK cluster?
 Please visit the [ACP-Kube](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube) repo on Gitlab to create a namespace.
 
-+ Clusters are defined by the folders within the `namespaces/` directory. This directory contains all defined namespaces for each given cluster.
++ Clusters are defined by the folders within the `policies/` directory, with each cluster having a subdirectory labelled `namespaces`, this directory contains all defined namespaces for each given cluster.
 
 + Namespaces should be defined in the following format within this repository:
-`namespaces/<cluster-name>/<namespace.yaml>`
+`policies/<cluster-name>/namespaces/<namespace.yaml>`
```
Comment (vinn946) [inline how-to-docs/namespace-how-to.md]: i think the directory path is wrong, should be `policies/clusters/<cluster-name>/namespaces/<namespace.yaml>`
e.g. `acp-kube/policies/clusters/acp-ops/namespaces/drone.yaml`

File: how-to-docs/namespace-how-to.md (hunk 2 above)
Comment (sjletcher) [reply how-to-docs/namespace-how-to.md]: does this need to be policies/clusters/bar/namespaces/foo.yaml

**Comment by tasharnvb** [comment]
LGTM

**Pull request merged**